backend/services/reports_service.py

```
@@ -2,50 +2,65 @@
+                birth_dt.day,
+                birth_dt.hour + birth_dt.minute / 60,
+            )
+            sun_long = swe.calc_ut(jd, swe.SUN)[0]
+            moon_long = swe.calc_ut(jd, swe.MOON)[0]
+            return {"sun_longitude": sun_long, "moon_longitude": moon_long}
+        else:
+            # Fallback calculation
+            day_of_year = birth_dt.timetuple().tm_yday
+            sun_long = (day_of_year / 365.25 * 360) % 360
+            moon_long = ((day_of_year * 13.2) + birth_dt.day * 13.2) % 360
+            return {"sun_longitude": sun_long, "moon_longitude": moon_long}
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)